release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -68,6 +68,22 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-ibm-power"]
 === IBM Power
 
+[id="ocp-4-12-images"]
+=== Images
+
+A new import value, `importMode`, has been added to the to the `importPolicy` parameter of image streams. The following fields are available for this value: 
+
+* `Legacy`: `Legacy` is the default value for `importMode`. When active,the manifest list is discarded, and a single sub-manifest is imported. 
+The platform is chosen in the following order of priority: 
++
+. Tag annotations
+. Control plane arch/OS 
```

Comment:
Suggestion: Spell out architecture (?) and remove slash to comply with IBM style (use "and" or "or").  (Unless "arch/OS" is the normal name for this.)